doc/man1/openssl-dhparam.pod.in

```diff
@@ -57,8 +57,6 @@ as the input filename.
 
 =item B<-dsaparam>
 
-This option is deprecated.
```

Comment:
Please remove the deprecation notice for -dsaparam also below in HISTORY.

Reply:
Fixed.